packages/web-components/src/components/cbp-notice/cbp-notice.tsx

```diff
@@ -0,0 +1,32 @@
+import { Component, Host, Prop, h } from '@stencil/core';
+
+@Component({
+  tag: 'cbp-notice',
+  styleUrl: 'cbp-notice.scss',
+
+})
+export class CbpNotice {
+
+  /** Optionally specifies a card color (different from the default color) based on predefined design token values. */
```

Comment:
I feel like this should default to "info". Can you check some other components with the color prop and see if they default?

Reply:
updated for default as 'info'. Looks like we might have some inconsistency on default's for color prop, looks like card doesn't have defaults but button does. might want to make a tech debt ticket or something on that later